lambda_handlers/handler.py

```diff
@@ -11,11 +13,12 @@ def handle(self, event, context):
         :param context:
         :return dict:
         """
+        print("EVENT:")
+        print(json.dumps(event))
         try:
             return self._handle(event, context)
         except Exception as e:
-            e.message = 'Bad Request: {0}'.format(e.message)
-            raise
+            raise EnvironmentError('Bad Request: {}'.format(e.message))
```

Comment:
This is actually why we have this global handle() function, so the API Gateway gets the same response when an exception is thrown.